cartridges/LINK_adyen/cartridge/client/default/js/adyenSecureFields.js

```diff
@@ -0,0 +1,49 @@
+    const configuration = {
+        locale: 'en_US', // Defaults to en_US
```

Comment:
Can you retrieve from the platform the users locale maybe? (It's nicer if you pass the default from the platform)